.github/actions/testagent/logs/action.yml

```diff
@@ -15,3 +15,17 @@ runs:
           docker-compose logs testagent
         fi
       shell: bash
+    - uses: actions/checkout@v2
+    - run: |
+        RESPONSE=$(curl -s -w "\n%{http_code}" -o response.txt http://127.0.0.1:9126/test/trace_check/failures)
```

Comment:
Should this and other scripts be moved to JS files in the `scripts` folder? This would make it easier to test them locally.